.github/workflows/main.yml

```diff
@@ -80,6 +83,13 @@ jobs:
         with:
           arch: ${{ env.MODEL == '64' && 'x64' || 'x86' }}
 
+      # NOTE: Linker ICEs with Xcode 15.0.1 (default version on macos-13)
+      # * https://issues.dlang.org/show_bug.cgi?id=24407
+      # Remove this step if the default gets changed to 15.1 in actions/runner-images.
+      - name: 'macOS 13: Switch to Xcode v15.1'
+        if: matrix.os == 'macos-13'
+        run: sudo xcode-select -switch /Applications/Xcode_15.1.app
+
```

Comment:
Do we trigger this in this repo?

Reply:
Sure; the steps are the same as for the DMD CI jobs. Both master/stable branches of Phobos are currently red, e.g., https://github.com/dlang/phobos/actions/runs/8118656250/job/22193306501.

Reply:
The only test I'm aware of that triggers the Xcode 15.0 bug is one that instantiates >32k templates (See the test case in the linked issue).

So unless phobos does that too, I think we can safely omit this step. :-)

Reply:
Phobos CI here runs the same tests as DMD. So if the problem is a test in `compiler/tests`, then it's tested by Phobos CI here too.

Reply:
Hmm, that really shouldn't be the case, but OK.